packages/browser/src/transports/base.ts

```diff
@@ -15,6 +15,7 @@ const CATEGORY_MAPPING: {
   event: 'error',
   transaction: 'transaction',
   session: 'session',
+  sessions: 'sessions',
```

Comment:
Maybe it isn't, its just that the server expects type `sessions` for request mode sessions and the payload for `sessions` (request mode sessions ) looks different from `session` (actual session) so thought it might be better to rate limit them differently but does not have to .. what do you think?